portal/jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   collectCoverageFrom: ['server/controllers/**/*.{js,}', 'server/routes/**/*.{js,}', 'server/helpers/**/*.{js,}', 'scripts/**/*.{js,}'],
   coverageDirectory: 'generated_reports/coverage/unit',
-  testMatch: ['**/*.test.js', '**/*.component-test.js'],
+  testMatch: ['**/getOneIndexedMonth.test.js'],
```

Comment:
Looks like this needs to be reverted?

Reply:
fwiw for unit tests I use the 'jest runner' package in vscode to run these tests easily. saves having to do this kind of thing - note it'll only pick up file called .test.js so component tests and api-tests I will rename to get jest runner to pick up.

Having written this down I'm sure you can configure the extension to not have to do that but I've never bothered